### PR TITLE
1317: Add higlighting to submenu items

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts [hunk 1]
```diff
@@ -77,12 +83,31 @@ export const getMenuItemStyles = ({ theme }: WfoTheme) => {
...
     const selectedSubMenuItem = css({
         ...baseSubMenuStyles,
+        height: `${theme.base * 2.25}px`,
+
+        backgroundColor: toSecondaryColor(theme.colors.mediumShade),
+        borderRadius: theme.border.radius.medium,
         fontWeight: theme.font.weight.medium,
         color: theme.colors.darkestShade,
+        marginLeft: `${theme.base / 4}px`,
+        paddingLeft: `${theme.base / 2}px`,
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts:93]: Think size.xs (currently 4px) would be a more safe way

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts [hunk 2]
```diff
@@ -77,12 +83,31 @@ export const getMenuItemStyles = ({ theme }: WfoTheme) => {
...
+        ':after': {
+            content: "''",
+            top: `${theme.base}px`,
+            left: '0px',
+            width: '4px',
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts:98]: size.xs can be used here

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts [hunk 3]
```diff
@@ -22,6 +22,12 @@ export const getMenuStyles = ({ theme }: WfoTheme) => {
         '.euiSideNavItem--branch': {
             '&:after': {
                 backgroundColor: theme.colors.lightShade,
+                height: '100%',
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts:25]: Do not forget to add changset in the PR

**Review by ricardovdheijden — APPROVED** [review]

**Review by Georgi2704 — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts:93]: Same

File: packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts (hunk 3 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoSidebar/styles.ts:25]: Done

**Pull request merged**